packages/grafana-ui/src/components/Select/_Select.scss

```diff
@@ -217,3 +220,7 @@ $select-input-bg-disabled: $input-bg-disabled;
 .select-button {
   display: flex;
 }
+
+.gf-form-select-box__img-value-hidden {
```

Comment:
where is this class set?

Reply:
It was added when I was experimenting with pure CSS solution only. Did the same thing the other way, already cleaned it up.